src/common/AndroidLauncher.ts

```diff
@@ -62,13 +75,7 @@ export class AndroidLauncher {
             })
             .then((actualPort) => {
                 emulatorPort = actualPort;
-                CommonUtils.startCliAction(
-                    `Launching`,
-                    `Waiting for device ${emuName} to boot`
-                );
-                return AndroidUtils.waitUntilDeviceIsReady(emulatorPort);
-            })
-            .then(() => {
```

Comment:
`AndroidUtils.startEmulator()` now takes a param (defaulting to true) indicating whether it should wait for the simulator and will internally call `waitUntilDeviceIsReady` so no need to do this here anymore.